srcpkgs/png++/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'png++'
+pkgname=png++
+version=0.2.9
+revision=1
+noarch=yes
+build_style=gnu-makefile
+make_build_target="docs"
+make_use_env=yes
+hostmakedepends="doxygen"
+makedepends="libpng-devel"
+short_desc="C++ wrapper for linpng library"
+maintainer="Alex Childs <misuchiru03+void@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://www.nongnu.org/pngpp"
+distfiles="${NONGNU_SITE}/releases/pngpp/${pkgname}-${version}.tar.gz"
```

Comment:
You don't need two templates for the same thing.

Reply:
Right. I had added `png++{,-doc}` and failed to remove the older `png++-devel{,-doc}`